### PR TITLE
Address some feedback from incoming R CMD check

### PR DESCRIPTION
**Pull request by Bisaloo**

### CODE REVIEW

**Review by pratikunterwegs — CHANGES_REQUESTED** [review]
Thanks @Bisaloo, CI builds fail as `RcppEigen` has been removed from `Depends`, so would ask to put that back. Otherwise looks alright to me on the whole.

Could you also clarify which of the tasks in #96 are addressed here, and mark them as done in the issue?

File: DESCRIPTION [hunk 1]
```diff
@@ -44,8 +44,7 @@ Suggests:
 Config/testthat/edition: 3
 License: MIT + file LICENSE
 Imports:
-    Rcpp,
-    RcppEigen
+    Rcpp
```
Comment (pratikunterwegs) [inline DESCRIPTION:47]:
```suggestion
    Rcpp,
    RcppEigen
```

Comment (Bisaloo) [reply DESCRIPTION:47]: I removed it from `Imports` on purpose as R CMD check flagged it should only be in `LinkingTo`. Indeed, it's never used in the R code, but only in the Rcpp code.

**Review by joshwlambert — APPROVED** [review]

**Comment by joshwlambert** [comment]
Not directly related to this PR, but is the `xml2` package used anywhere? Could potentially be removed from `Suggests`.

**Comment by Bisaloo** [comment]
> Not directly related to this PR, but is the xml2 package used anywhere? Could potentially be removed from Suggests.

Good question, see the discussion [in this previous PR](https://github.com/epiverse-trace/finalsize/pull/36#discussion_r969802936).

**Pull request merged**